bin/ebuild-helpers/prepstrip

```diff
@@ -143,7 +143,7 @@ save_elf_debug() {
 	local x=$1
 	local inode_debug=$2
 	local splitdebug=$3
-	local y=${ED}usr/lib/debug/${x:${#D}}.debug
+	local y=${ED%/}/usr/lib/debug/${x:${#D}}.debug
```

Comment:
We need to fix `${x:${#D}}` to account for `${D}` without a trailing slash, maybe use `local d=${D%/}` and `${#d}`.

Reply:
Ai, my grep didn't catch `${#D` etc.